python/lsst/cp/pipe/ptc.py

```diff
@@ -406,19 +304,20 @@ def runDataRef(self, dataRefList):
                 doRealSpace = self.config.covAstierRealSpace
                 muDiff, varDiff, covAstier = self.measureMeanVarCov(exp1, exp2, region=amp.getBBox(),
                                                                     covAstierRealSpace=doRealSpace)
+                datasetPtc.rawExpTimes[ampName].append(expTime)
+                datasetPtc.rawMeans[ampName].append(muDiff)
+                datasetPtc.rawVars[ampName].append(varDiff)
+                datasetPtc.inputExpIdPairs[ampName] = expIds
```

Comment:
I don't think it's necessarily wrong, but doesn't this reset `datasetPtc.inputExpIdPairs[ampName]` on each loop through the `expPairs`?

Reply:
I took it out of the expPairs loop to set it once.